### ISSUE TITLE
How to do label smoothing with loss_binary_crossentropy

### ISSUE THREAD

**Issue by dslate1** [rstudio/keras#1087]
I've searched the Web but have been unable to figure out how to do "label smoothing" in conjunction with loss_binary_crossentropy in Keras.  I saw an example from the Python world, in which the binary_crossentropy function seems to have a parameter called "label_smoothing", but haven't found the corresponding parameter in the R API.  Does it exist, and, if not, can I get the desired effect anyway by changing the labels themselves from strict 0 or 1 to floating point numbers in between?

Thanks.

**Comment by dfalbel** [comment]
Indeed this argument was not available. I added in #1088.

While I don't merge, you can install from this branch using `devtools::install_github("rstudio/keras#1088")`.

You can also use the loss directly from the tf module. Eg: `tensorflow::tf$keras$losses$binary_crossentropy`.

**Comment by dslate1** [comment]
Thanks dfalbel.  I am trying tensorflow::tf$keras$losses$binary_crossentropy as you suggest.  It seems to be working, although I'm still figuring out how much smoothing, if any, is beneficial in my application.
